packages/core/src/lib/implementation/read-rc-file.integration.test.ts

```diff
@@ -37,6 +37,15 @@ describe('readRcByPath', () => {
     );
   });
 
+  it('should load the configuration using provided tsconfig', async () => {
+    await expect(
+      readRcByPath(
+        join(configDirPath, 'code-pushup.needs-tsconfig.config.ts'),
+        'tsconfig.base.json',
+      ),
+    ).resolves.toBeTruthy();
```

Comment:
This might still return empty places where the models have been used. I think it would be valuable to check the properties where the package was used (so `audits` and `runner`).